homeassistant/components/nuki/lock.py

```diff
@@ -68,10 +68,11 @@ def service_handler(service):
         DOMAIN, SERVICE_LOCK_N_GO, service_handler, schema=LOCK_N_GO_SERVICE_SCHEMA
     )
 
+    devices.extend([NukiOpenerEntitiy(opener) for opener in bridge.openers])
     add_entities(devices)
 
 
-class NukiLock(LockEntity):
+class NukiLockEntitiy(LockEntity):
```

Comment:
This looks like a typo.
Entitiy -> Entity

Reply:
Ahh! Thanks for catching that.